### PR TITLE
Week 2 blog

### PR DESCRIPTION
**Pull request by mmurray22**
This PR centers around the 'blog' feature of the website. The main files to inspect are the blog.html, blog_style.css, and blog_script.js. Other files with related functionality includes the index.html (where the feature need analysis will be the 'Blog' button) and script.js (where the feature needing analysis will be the 'blog()' function).

### CODE REVIEW

File: portfolio/src/main/webapp/blog.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <!-- Imported Libraries: Google's Material Design Library and Google's Fonts Library -->
+    <link href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+    <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
+    <title>MM Portfolio</title>
+    <link rel="stylesheet" href="blog_style.css">
+    <script src="blog_script.js"></script>
```
Comment (ccondit) [inline portfolio/src/main/webapp/blog.html:11]: Do these resources need a leading slash to work correctly in the container?

File: portfolio/src/main/webapp/blog.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  </head>
+  <body>
+    <div class="mdc-touch-target-wrapper" style="display: block;">
+	  <button class="mdc-button mdc-button--outlined" onclick="return_home()" style="color:white;">  
+	    <span class="mdc-button__ripple">Return</span>
+	  </button>
+    </div>
+    <div id="my-profile">
```
Comment (ccondit) [inline portfolio/src/main/webapp/blog.html]: nit - update the id to reflect what's actually in the `div`

File: portfolio/src/main/webapp/blog.html [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+      <h1>Blog</h1>
+      <div id="intro">
+      	<ul style="list-style-type:none;">
+           <li>Stanford Class of 2022</li> 
+           <li>Major: Computer Science</li>
+           <li>STEP Intern @ Google</li>
+	    </ul>
```
Comment (ccondit) [inline portfolio/src/main/webapp/blog.html]: Just checking: do you want the intro content on the blog page, as well?

**Review by ccondit — CHANGES_REQUESTED** [review]
Can you scope this PR to only adding:
* blog.html
* blog_script.js
* blog_style.css

File: portfolio/src/main/webapp/blog.html (hunk 3 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/blog.html]: Nope! I was using it as filler, but I can remove it! Thanks!

File: portfolio/src/main/webapp/blog.html (hunk 2 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/blog.html]: Noted! Thanks!

File: portfolio/src/main/webapp/blog.html (hunk 1 above)
Comment (mmurray22) [reply portfolio/src/main/webapp/blog.html:11]: I don't believe so! I have code both in the stylesheet/script that run in the blog :)

**Pull request closed without merging**